app/helpers/users_helper.rb

```diff
@@ -0,0 +1,7 @@
+module UsersHelper
+  def gravatar_for(user, size: 80)
```

Comment:
`80` em khai báo giá trị này vào file settings em ơi

nếu em chưa có file `settings` thì em tìm hiểu `gem config in rails` trước nhé